docs/running/index.rst

```diff
@@ -21,6 +21,7 @@ A quick start guide to run TARDIS can be found
     :maxdepth: 2
     :caption: Running TARDIS and interactivity
 
+    Widget_Documentation
```

Comment:
For the sake of ease in typing/finding files (esp. in linux file system), we usually keep filenames all in small letters, with hyphen for spaces. (It's ok for now, but I just wanted to inform you this for your future development projects).

Reply:
Sure. I understand that